tracker/gioui/tracker.go

```diff
@@ -89,9 +89,16 @@ func NewTracker(model *tracker.Model) *Tracker {
 		Model: model,
 
 		filePathString: model.FilePath().String(),
+		preferences:    NewPreferences(),
 	}
 	t.Theme.Shaper = text.NewShaper(text.WithCollection(fontCollection))
 	t.PopupAlert = NewPopupAlert(model.Alerts(), t.Theme.Shaper)
+	if t.preferences.YmlError != nil {
```

Comment:
This is fine for now. In future, and even for the keybindings, I'm thinking of loading them every time a new tracker is created instead of init(). Because:

In the VST, init is called only once, when the DLL is loaded. But the DLL stays loaded even if you close the plugin. So, to make new config take effect, you have to close the entire DAW.